src/harness/BenchmarkDotNet.Extensions/CommandLineOptions.cs

```diff
@@ -30,25 +30,21 @@ public static List<string> ParseAndRemoveIntParameter(List<string> argsList, str
             return argsList;
         }
 
-        public static List<string> ParseAndRemoveStringParameter(List<string> argsList, string parameter, out string parameterValue)
+        public static List<string> ParseAndRemoveStringsParameter(List<string> argsList, string parameter, out List<string> parameterValue)
         {
             int parameterIndex = argsList.IndexOf(parameter);
-            parameterValue = null;
+            parameterValue = new List<string>();
 
-            if (parameterIndex != -1)
+            if (parameterIndex + 1 < argsList.Count)
             {
-                if (parameterIndex + 1 < argsList.Count)
+                while (parameterIndex + 1 < argsList.Count && !argsList[parameterIndex + 1].Contains("-"))
```

Comment:
nit: `StartsWith` could be potentially more future-proof in case somebody sends an argument with `-` in the value (some path for example?)

```suggestion
                while (parameterIndex + 1 < argsList.Count && !argsList[parameterIndex + 1].StartsWith("-"))
```

Reply:
Good idea